docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -81,6 +81,34 @@ The path to the Teku data directory. The default directory is OS dependent:
 
 The default Docker image location is `/root/.local/share/teku`.
 
+### data-storage-archive-frequency
+
+```bash tab="Syntax"
+--data-storage-archive-frequency=<NUMBER>
+```
+
+```bash tab="Command Line"
+--data-storage-archive-frequency=1028
+```
+
+```bash tab="Environment Variable"
+TEKU_DATA_STORAGE_ARCHIVE_FREQUENCY=1028
+```
+
+```bash tab="Configuration File"
+data-storage-archive-frequency: 1028
+```
+
+Set the frequency (in slots) at which to store finalized states to disk. Defaults to 2048.
+
+This option is ignored if [`--data-storage-mode`](#data-storage-mode) is set to `prune`.
+
+!!! note
+    Lower archive frequencies have a potentially higher overhead for retrieving finalized states
+    since more states may need to be regenerated to get to the requested state.
+    
+    Higher archive frequencies increase the disk space usage.
```

Comment:
dont hate me.... 'higher archive frequency' is a lower number, is that confusing?

`--data-storage-archive-frequency=1` - store every finalized state
`--data-storage-archive-frequency=2048` - store 1 in 2048 states.

Reply:
I feel it is. Was going to add something to clarify. But wasn't sure how to phrase it :)